client/my-sites/customer-home/components/full-screen-launchpad.tsx

```diff
@@ -59,41 +70,65 @@ export const FullScreenLaunchpad = ( { onClose }: { onClose: () => void } ): JSX
 		dispatch( requestSite( siteId ) );
 	};
 
+	const {
+		data: { checklist },
+	} = useSortedLaunchpadTasks( siteSlug, checklistSlug, launchpadContext );
+
 	if ( isDismissed ) {
 		return null;
 	}
 
-	const isAllTasksCompleted = hasChecklist && numberOfSteps > 0 && completedSteps === numberOfSteps;
+	const launchSiteTask = checklist?.find( ( task: Task ) =>
+		[ 'site_launched', 'blog_launched', 'link_in_bio_launched' ].includes( task.id )
+	);
+
+	const isAllTasksCompleted =
+		hasChecklist &&
+		numberOfSteps > 0 &&
+		completedSteps >= numberOfSteps - ( launchSiteTask ? 1 : 0 );
 
 	return (
-		<div css={ { display: 'flex', flexDirection: 'column', alignItems: 'center' } }>
-			<div className="is-launchpad-first" css={ { width: '100%' } }>
-				<div className="customer-home-launchpad customer-home__card is-small-hero">
-					<div className="customer-home__launchpad-header">
-						<CircularProgressBar
-							size={ 40 }
-							enableDesktopScaling
-							numberOfSteps={ numberOfSteps }
-							currentStep={ completedSteps }
-						/>
-						<h2>
-							{ ! isAllTasksCompleted ? __( "Let's get started!" ) : __( "You're all set!" ) }
-						</h2>
-						<span>{ ! isAllTasksCompleted && launchpadTitle }</span>
-					</div>
-					{ isAllTasksCompleted && <ConfettiAnimation /> }
-					<Launchpad
-						siteSlug={ siteSlug }
-						checklistSlug={ checklistSlug }
-						launchpadContext={ launchpadContext }
-						onSiteLaunched={ onSiteLaunched }
-						highlightNextAction
+		<div className="is-launchpad-first" css={ { width: '100%' } }>
+			<div
+				className={ `customer-home-launchpad customer-home__card is-small-hero ${
+					isAllTasksCompleted ? 'all-tasks-completed' : ''
+				}` }
+			>
+				<div className="customer-home__launchpad-header">
+					<CircularProgressBar
+						size={ 40 }
+						enableDesktopScaling
+						numberOfSteps={ numberOfSteps - ( launchSiteTask ? 1 : 0 ) }
+						currentStep={ completedSteps }
 					/>
+					<h2>{ ! isAllTasksCompleted ? __( "Let's get started!" ) : __( "You're all set!" ) }</h2>
+					<span>{ ! isAllTasksCompleted && launchpadTitle }</span>
+				</div>
+				{ isAllTasksCompleted && <ConfettiAnimation /> }
+				<Launchpad
+					siteSlug={ siteSlug }
+					checklistSlug={ checklistSlug }
+					launchpadContext={ launchpadContext }
+					onSiteLaunched={ onSiteLaunched }
+					highlightNextAction
+				/>
+				<div className="launchpad-actions">
+					{ launchSiteTask && isAllTasksCompleted && (
+						<Button
+							onClick={ onSiteLaunched }
+							className="launchpad-site-launch"
+							variant="primary"
+							isBusy={ isLaunching }
+							disabled={ isLaunching }
+						>
+							{ launchSiteTask?.title }
+						</Button>
```

Comment:
We can't override Launchpad's Launch site button behavior, like adding the `isBusy` state, so I decided to reimplement it as a dedicated button here and hide the Launchpad task one.